extensions/multiply-divide/README.md

```diff
@@ -0,0 +1,62 @@
+# General design
+
+**Extension State: Under Development**  
+**Requires: Base, VWI, CP2.0**  
+**CPUID Bit: CP2.3**
+
+# Overview
+
+This extension adds multiplication, division, and remainder instructions. 
+
+# Added Instructions
+
+These instructions are in the expanded calculation opcode section of instructions.
+
+A `U` at the front of an instruction indicates that it treats its operands as unsigned.
+An `S` indicates that the operands are treated as signed.
+
+### Opcode table
+
+| `C CCCC CCCC` | NAME     | Operation                                  | Flags  | Comment |
+|---------------|----------|--------------------------------------------|--------|---------|
+| `0 0001 0000` | `UDIV`   | `A ← A / B`                                | None   | (1)     |
+| `0 0001 0001` | `SDIV`   | `A ← A / B`                                | None   | (1)     |
+| `0 0001 0010` | `UREM`   | `A ← A % B`                                | None   | (1) (2) |
+| `0 0001 0011` | `SREM`   | `A ← A % B`                                | None   | (1) (2) |
+| `0 0001 0100` | `UMUL`   | `A ← low(A × B)`                           | `CV`   | (3) (5) |
+| `0 0001 0101` | `SMUL`   | `A ← low(A × B)`                           | `CV`   | (3) (6) |                                          
+| `0 0001 0110` | `UHMUL`  | `A ← high(A × B)`                          | None   | (4)     |
+| `0 0001 0111` | `SHMUL`  | `A ← high(A × B)`                          | None   | (4)     |                                          
```

Comment:
Does `None` for flags mean that the flags are not affected or that they're in an undefined state? Maybe we need to do a re-factor to the rest of the code base to make it explicit which instructions make all flags undefined and which ones don't touch any flags?

Reply:
There are no instructions that make all flags undefined. That wouldn't be useful.

Reply:
We should probably be consistent with the rest of the spec then. (even if we do a different PR to change it all to None instead of being blank)